### PR TITLE
chapter 4

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: Chapter_4/README.md [hunk 1]
```diff
@@ -1,30 +1,43 @@
...
-# Setting up DVC
+As you might have already noticed, there are some similarities between W&B and MLFlow. However, in the case of Mlflow, every experiment is linked with a set of Git commits. Git does not prevent us from saving datasets, but it shows many limitations when the datasets are large, even with an extension built for large files (Git-LFS). Thus, MLflow is commonly combined with [DVC] (https://dvc.org/), an open-source version control system that solves Git limitations.
+
+Both MLFlow and DVC can be installed through pip.
 ```
+pip install mlflow
 pip install dvc
 ```
 
+In the notebook, [MLFlow_dvc.ipynb](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/blob/main/Chapter_4/MLFlow_dvc.ipynb), we describe how to use MLFlow and DVC for tracking a model training.
```
Comment (ljj7975) [inline Chapter_4/README.md]: @tpalczew can you add sample notebook for mlflow and dvc?

Comment (tpalczew) [reply Chapter_4/README.md]: Added

**Comment by tpalczew** [comment]
Are we good here? @ljj7975  
Check what is in the master and if so please close.

**Review by tpalczew — APPROVED** [review]
+1

**Pull request merged**